src/supporting-data/taxonomy/components/entry/ChildNavigation.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link } from 'react-router-dom';
+import { Button } from 'franklin-sites';
+
+import { pluralise } from '../../../../shared/utils/utils';
+import {
+  getEntryPath,
+  LocationToPath,
+  Location,
+} from '../../../../app/config/urls';
+
+import { TaxonomyAPIModel } from '../../adapters/taxonomyConverter';
+import { Namespace } from '../../../../shared/types/namespaces';
+
+import styles from './styles/child-navigation.module.css';
+
+// Match the "ExpandableList" default count
+export const CHILDREN_TO_DISPLAY = 5;
+
+const ChildNavigation = ({
+  taxonId,
+  childTaxons,
+  total,
+}: {
+  taxonId: number;
+  childTaxons: TaxonomyAPIModel[];
+  total: number;
+}) => {
+  const remainingChildren = total - CHILDREN_TO_DISPLAY;
+  return (
+    <ul className="no-bullet">
+      {childTaxons.map((child) => (
+        <li key={child.taxonId}>
+          <Link to={getEntryPath(Namespace.taxonomy, child.taxonId)}>
+            {child.commonName || child.scientificName || child.taxonId}
+          </Link>
+        </li>
+      ))}
+      <li>
+        <Button
+          to={{
+            pathname: LocationToPath[Location.TaxonomyResults],
+            search: `query=parent:${taxonId}`,
+          }}
+          element={Link}
+          variant="tertiary"
+          className={styles['no-margin']}
+        >
+          {remainingChildren > 0
+            ? `${remainingChildren} more direct ${pluralise(
```

Comment:
1. I like the word `browse` you've used in

- "Browse all direct children"
- "Browse all descendants"

because it lets me know that I'm going to be taken to a new page of results when I click the button. What do you think about also having it for the case of `remainingChilren > 0` too? Eg

`22 more direct children`  --> `Browse 22 more direct children`

2. What do you think about just showing `Browse all direct children (27)` (eg not subtracting what is visible from the count) as in a sense we are browsing all of the direct children (and not just those that aren't currently visible).

Reply:
I like option 2. Less logic, less cognitive overhead for the user (now it would be "so... 22 + the ones I see, let's count...").
I will do that